client/app/dedicated/server/bandwidth-legacy/order-public/dedicated-server-bandwidth-order.controller.js

```diff
@@ -148,10 +148,8 @@ angular.module('App').controller('ServerOrderLegacyBandwidthCtrl', ($scope, $sta
 
         bcUrl = durations.url;
       },
-      (data) => {
-        $scope.resetAction();
-        _.set(data, 'data.type', 'ERROR');
-        $scope.setMessage($translate.instant('server_order_bandwidth_error'), data.data);
+      () => {
```

Comment:
catch ?

Reply:
I tried to limit the changes to this legacy component to as few as possible, but I keep the refactoring in mind